apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -63,13 +66,22 @@ defmodule OMG.WatcherInfo.DB.Block do
   def get(blknum) do
     query =
       from(
-        __MODULE__,
+        block in base_query(),
         where: [blknum: ^blknum]
       )
 
     DB.Repo.one(query)
   end
 
+  def base_query() do
```

Comment:
Can you add a test for this new function into `DB.BlockTest`?

Reply:
Done de5e2c7